.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -0,0 +1,28 @@
+---
+name: User story
+about: New feature or enhancement
+title: ''
+labels: user story
+assignees: ''
+
+---
+
+**As a** [description of user]
+**I want to** [functionality]
+**So that** [benefit]
+
+**Context:**
+Describe the context and any relevant information about this user story
+
+**Acceptance Criteria:**
+**Given** [how things begin]
+**When** [action taken]
+**Then** [outcome]
+
+**Scope**
```

Comment:
let's add a section that says what is the component:
- Server
- Dashboard
- etc

Reply:
done